nova_vm/src/engine/bytecode/vm.rs

```diff
@@ -1003,6 +1016,56 @@ impl Vm {
                 let w = vm.result.take().unwrap();
                 initialize_referenced_binding(agent, v, w)?;
             }
+            Instruction::InitializeVariableEnvironment => {
+                let num_variables = instr.args[0].unwrap();
+                assert!(instr.args[1].unwrap() <= 1);
```

Comment:
**thought:** I was surprised this couldn't be done with `bool::try_from(u16_value).unwrap()` but apparently that's not a thing.